content/ops/metrics.md

```diff
@@ -5,6 +5,8 @@ menu:
 title: Metrics
 ---
 
+***This page is out of date.***
```

Comment:
Not really sure _how much_ of this page is no longer accurate, would help for someone that knows more than me to go through it.

https://github.com/18F/cg-docs/issues/447